README.md

```diff
@@ -1,2 +1,34 @@
-# Portfolio-Mobile-Version-
-Setup a mobile version portfolio to display my projects and skills
+# Portfolio-Mobile-Version
+Setup a mobile version Portfolio: Mobile Version Projects PR to display my projects and skills
+
+Hi, I simple coded the mobile version with the help of flexbox and figma features. It is a structure where we decorate and furnishe further. Develop the responsive one, It's a Portfolio: Mobile Version projects display. It helps the users to understand our services better. It also helps users to understand now a days features and techs.
+
+Built With Major languages Frameworks Technologies used Html, Css, linters.
+Grid,
+Flexbox,
+Hover
+
+To get a local copy up and running follow these simple example steps.
+
+Prerequisites 
+Setup 
+Install 
+Usage 
+Run 
+tests
+Deployment 
+Authors 
+👤 Author
+
+GitHub: @ch-ahtesham-ul-haq Twitter: @chaury39 
+LinkedIn: https://www.linkedin.com/in/ch-ahtesham-ul-haq-b5318b152/
+
+🤝 Contributing Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support Give a ⭐️ if you like this project!
+
+Acknowledgments Hat tip to anyone whose code was used Inspiration etc 
+📝 License
+ This project is MIT licensed.
```

Comment:
Hi, even though you used the content from the provided README template, you didn't use the markdown syntax to style the file. Kindly update it with the same. If you have any difficulty, refer to [this](https://www.markdownguide.org/cheat-sheet/) article

Reply:
Okay Thank You So Much